### PR TITLE
WP add annotation tool blog.

### PR DESCRIPTION
**Pull request by willpoulett**
# Description

❓**What**: Added a new blog in the articles section.

🧠**Why?**: To add content.

👨‍💻**How?**: Cadded a new md file in articles section, added gifs and images in images folder. 

# Checklist:
Have checked for the following:
- [X ] The website still builds correctly, and you can view it using `mkdocs serve`.
- [ X] There are no new "warnings" from mkdocs
- [ X] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))? (**need to make a new one specific to NHSE Data Science**)
- [ X] Spelling errors
- [ X] Consistent capitalization
- [ X] Consistent numbers
- [ X] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [ X] Code snippets don't work
- [ X] Images not working
- [ X] Links not working

## Where it was tested

- Github Codespaces - 2-core, 4GB RAM, 32GB hard drive

### CODE REVIEW

File: docs/articles/posts/20240807_annotation_tools.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+We want a tool that:
+
+- Is easy for a user to setup and use.
+- Allows users to label new entities.
+- Gets live feedback from the NER model.
+
+There were two approaches we took to develop an annotation tool.
+
+### DisplaCy + ipyWidgets
+
```
Comment (amaiaita) [inline docs/articles/posts/20240807_annotation_tools.md:70]:
```suggestion
### DisplaCy and ipyWidgets

```

File: docs/articles/posts/20240807_annotation_tools.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+<figure markdown>
+![Example annotation gif using ipywidgets](../../images/annotation_tools_blog/ipywidgets_example.gif)
+<figcaption>Figure 2: An example of the ipyWidgets and DisplaCy labelling application. All clinicial notes are synthetic. </figcaption>
+</figure>
+
+First, we used [DisplaCy](https://spacy.io/usage/visualizers/), [ipyWidgets](https://github.com/jupyter-widgets/ipywidgets/blob/main/docs/source/examples/Index.ipynb) and a NER model of choice to generate an interactive tool that works inside Jupyter notebooks. DisplaCy is a visualiser integrated into the SpaCy library which allows you to easily visualise labels. Alongside ipyWidgets, a tool that allows you to create interactive widgets such as buttons, we created an interface which allowed a user to go through reviews and add new entities.
```
Comment (amaiaita) [inline docs/articles/posts/20240807_annotation_tools.md:76]:
```suggestion
First, we used [DisplaCy](https://spacy.io/usage/visualizers/), [ipyWidgets](https://github.com/jupyter-widgets/ipywidgets/blob/main/docs/source/examples/Index.ipynb), and a NER model of choice to generate an interactive tool that works inside Jupyter notebooks. DisplaCy is a visualiser integrated into the SpaCy library which allows you to easily visualise labels. Alongside ipyWidgets, a tool that allows you to create interactive widgets such as buttons, we created an interface which allowed a user to go through reviews and add new entities.
```

oxford comma!!

File: docs/articles/posts/20240807_annotation_tools.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+One of the main advantages of this method is that everything is inside a jupyter notebook. The entity names you want to extract come straight from the experiment parameters, so if you used this in the same notebook as the rest of your pipeline the entitiy names could be updated automatically from the labelling tool. This would allow easy integration into a user workflow.
```
Comment (amaiaita) [inline docs/articles/posts/20240807_annotation_tools.md:78]: Jupyter is a brand name so should be capitalised

File: docs/articles/posts/20240807_annotation_tools.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
+---
+title: Investigating Annotation Tools for Named Entity Recognition
+authors: [willpoulett]
+date: 2024-08-07
+categories:
```
Comment (amaiaita) [inline docs/articles/posts/20240807_annotation_tools.md:5]:
```suggestion
date: 2024-07-08
categories: 
```

**Review by amaiaita — COMMENTED** [review]
just a couple of small changes - looks good !

File: docs/articles/posts/20240807_annotation_tools.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    - LLMs
+    - Python
+    - Annotation Tools
+links:
+    - TBC
+slug: annotation-tools
+description: >
+    There are various tools for annotating and exploring free text data for Named Entity Recognition. The author explores some of these tools and discusses his experiences. 
+---
+
```
[the remark this answers is not in the document]
Comment (willpoulett) [reply docs/articles/posts/20240807_annotation_tools.md:16]: Done at the bottom :)

[the remark this answers is not in the document]
Comment (willpoulett) [reply docs/articles/posts/20240807_annotation_tools.md:16]: Realised it is actually on line 1!

**Review by amaiaita — APPROVED** [review]

**Pull request merged**